app/components/PreHeader.tsx

```diff
@@ -5,11 +5,11 @@ import SocialLinks from "./SocialLinks";
 
 const PreHeader = () => {
   return (
-    <nav className="hidden lg:flex w-full bg-zinc-100 px-6 text-black">
+    <nav className="hidden lg:flex w-full bg-grey-20 px-6 text-black">
```

Comment:
Should this be `200` rather than `20`?

Reply:
I've overwritten tailwind colours to only have ones the site will use, rather than extend. Figma has it as Grey 20.

Do you have a preference on colour names? Stick with 'primary' e.t.c. this or rename to using 'teal-light' and 'red-dark' for example?